Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
2016-04-07 16:59 GMT+02:00 Rainer Gerhards <rgerhards@hq.adiscon.com>:
> 2016-04-07 16:56 GMT+02:00 Michael Biebl <notifications@github.com>:
>>
>> In Makefile.am:
>>
>> > @@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
>> >
>> >  libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined
>> > @JSON_BSYMBOLIC_LDFLAGS@ \
>> >   -export-symbols-regex '^fjson_.*'
>> > -libfastjson_la_CPPFLAGS = -Werror
>>
>> ie., let's just drop the usage of Werror until we re-introduce it via
>> AX_COMPILER_FLAGS
>>
>>
> OK, that works. But I would like to check on the internal headers before
> merging. It's easy if I complete my current task, but mixing this merge and
> the tasks costs me much more coordination over the projects. So bear with me
> for a couple of hours. (As I wrote in the overall issue, that commit breaks
> rsyslog build).
>

In other words: I don't think it doesn't really matter if we merge
today, tomorrow or monday, but it eases my life considerably if we do
not merge immediately.